core/shared/src/main/scala/zio/stm/STM.scala

```diff
@@ -102,13 +102,31 @@ object STM {
     }
 
   /**
-   * Creates an STM effect from an `Either` value.
+   * @see See [[zio.stm.ZSTM.fromEither]]
    */
   def fromEither[E, A](e: => Either[E, A]): STM[E, A] =
     ZSTM.fromEither(e)
 
   /**
-   * Creates an STM effect from a `Try` value.
+   * @see See [[zio.stm.ZSTM.fromFunction]]
+   */
+  def fromFunction[R, A](f: R => A): ZSTM[R, Nothing, A] =
```

Comment:
Do we want to constrain the types on these? Normally here and like in `IO.scala` we would constrain the `R` type to be `Any`, which admittedly is not particularly useful but is consistent with elsewhere and honors the contract that a method on the `STM` companion object is going to give you an instance of `STM`.

Reply:
Absolutely, fixed.